### PR TITLE
Format app context files with the Eclipse xml formatter

### PR DESCRIPTION
**Pull request by aaime**
Same goal as https://github.com/geoserver/geoserver/pull/4663, but trying out a different tool.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or not applicable.**

For all pull requests:

- [ ] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [ ] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [ ] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by Continuous Integration after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by Continuous Integration after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

### CODE REVIEW

File: src/security/pom.xml [hunk 1]
```diff
@@ -1,59 +1,61 @@
...
+        <artifactId>geoserver</artifactId>
+        <version>2.19-SNAPSHOT</version>
+    </parent>
 
-  <groupId>org.geoserver</groupId>
-  <artifactId>gs-security</artifactId>
-  <packaging>pom</packaging>
-  <name>GeoServer Security Modules</name>
+    <groupId>org.geoserver</groupId>
+    <artifactId>s-security</artifactId>
```
Comment (bradh) [inline src/security/pom.xml:21]: Is this from the tooling or manual edits? Certainly looks a bit strange, and appears to be (part of) what is breaking the build.

Comment (aaime) [reply src/security/pom.xml:21]: Hum... wondering if the plugin is safe to parallel builds

Comment (aaime) [reply src/security/pom.xml:21]: Indeed I'm getting other failures as well. Reported here:
https://github.com/revelc/formatter-maven-plugin/issues

**Comment by aaime** [comment]
Reduced the scope to just formatting the application context files. Unfortunately, it's still not working in a parallel build. Leaving this as a reference for the revelc people to look at, if they so desire.

**Comment by aaime** [comment]
For reference, this command, getting most of the modules in the build, almost systematically fails on a AMD Ryzen 1700x, 8 core:

```
mvn formatter:format -nsu -T6 -Prelease,communityRelease
```

The build above involves over 200 modules. The failures involve different modules at each run.

**Pull request closed without merging**